CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 
 This document describes changes between each past release.
 
-3.0.1 (unreleased)
+3.0.1 (2016-05-20)
 ==================
 
 **Bug fixes**
```

Comment:
nitpick: We usually add a reference to the original PR/issue for each entry in the changelog